Controllers/Admin/TaskController.php

```diff
@@ -0,0 +1,157 @@
+<?php
+declare(strict_types=1);
+
+namespace Controllers\Admin;
+
+use Controllers\BaseController;
+use Models\TaskModel;
+use View\TaskView;
+
+/**
+ * @package Controllers\Admin
+ */
+class TaskController extends BaseController
+{
+    /**
+     * @var TaskModel
+     */
+    private $taskModel;
+
+    /**
+     * @var TaskView
+     */
+    private $taskView;
+
+    public function __construct()
+    {
+        $this->taskModel = new TaskModel();
+        $this->taskView = new TaskView();
+    }
+
+    /** Render admin panel by param
+     * @param array $param
+     */
+    public function formAction(array $param): void
+    {
+        $access = $this->taskModel->isAccess();
+        if ($access) {
+            switch ($param[0]) {
+                case 'delete' :
+                    $data = $this->taskModel->adminGetAllTasks($access);
+                    $options = ['title' => 'Удаление',
+                        'content' => 'admin/delete_task.phtml',
+                        'data' => $data];
+                    break;
+                case 'change' :
+                    $data = $this->taskModel->adminGetAllTasks($access);
+                    $options = ['title' => 'Изменить',
```

Comment:
i think u can create function named like getActionOptions that will return an array of needed data. instead of making a lot of code lines declaration an array this way